vars/neoDeploy.groovy

```diff
@@ -27,14 +27,17 @@ def call(parameters = [:]) {
         'warAction'
         ]
 
+    List generalConfigurationKeys = [
+        'neoHome'
```

Comment:
What is the reason for providing 'neoHome' in the generalConfiguration. To my understanding this property is specific to step 'neoDeploy'. Is it intended to use this property elsewhere?

In my understanding the semantics of the 'general' section in the config is to provide configuration which is valid across multiple steps. Am I in error with my understanding?

Reply:
Yes, it is intended to reuse it, I already told you in PR #75, that it is open for one week by the way, I don't understand why you begin reviewing this PR.

Reply:
??? It is a pull request and it is not labeled with "work in progress".
So what is wrong with reviewing it. In principle I think there should also no dependencies between pull requests since this messes up the review process

Reply:
We have a missunderstanding about "reuse" here. I do not mean re-use like "reuse a method in another piece of software". What I mean here is: is the neoHome property re-used by another step? If it is not re-used by another step it would be interesting to know the reason why we provide it in the general section (which is  - at least to my understanding - designed for keeping properties which are important across steps.)

In case we decide to re-factor the validation the new validation should IMO still query the step specific configuration when checking the prerequisites for a certain step.

Reply:
There is no misunderstanding, neoHome is reused by another step.

Reply:
By which step? If is is the validate step: I think that does not count. If we would take that into account each property would be general

Reply:
There is no validate step as far as I know. By the toolValidate step.